Modules/Module_Import/NII2BIDS/xASL_bids_PhoenixProtocolAnalyzer.m

```diff
@@ -14,198 +14,256 @@
+    sourcePar = addParToList('sWipMemBlock.tFree',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.alFree.__attribute__.size',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.alFree[0]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.alFree[1]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.alFree[2]',sourcePar, parIndex);parIndex = parIndex+1;
+	sourcePar = addParToList('sWipMemBlock.alFree[3]',sourcePar, parIndex);parIndex = parIndex+1;
+	sourcePar = addParToList('sWipMemBlock.alFree[4]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.alFree[5]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.alFree[6]',sourcePar, parIndex);parIndex = parIndex+1;
+	sourcePar = addParToList('sWipMemBlock.alFree[9]',sourcePar, parIndex);parIndex = parIndex+1;
+	sourcePar = addParToList('sWipMemBlock.alFree[10]',sourcePar, parIndex);parIndex = parIndex+1;
+	sourcePar = addParToList('sWipMemBlock.alFree[11]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.alFree[63]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.adFree.__attribute__.size',sourcePar, parIndex);parIndex = parIndex+1;
+	sourcePar = addParToList('sWipMemBlock.adFree[0]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.adFree[1]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.adFree[7]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.adFree[8]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.adFree[9]',sourcePar,parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.adFree[10]',sourcePar, parIndex);parIndex = parIndex+1;
+    sourcePar = addParToList('sWipMemBlock.adFree[13]',sourcePar, parIndex);parIndex = parIndex+1;
 
-    
-    %% Get the predefined parameters
+    % Get the predefined parameters
     sourcePar = getPhoenixParameters(sourcePar,parameterList,false);
     
-    %% Get xASL parameters
+    % Get xASL parameters
     sourcePar = convertCellArrayToStruct(sourcePar);
 
+	%% 2. Obtain basic information about the sequence
 	bidsPar = struct();
-    %% Analyze parameters                                                   % Exemplary dataset in ExploreASL flavor library
-	if ~isempty(strfind(sourcePar.tSequenceFileName,'SiemensSeq'))
-		bidsPar.SequenceType = 'Siemens'; % Not really a BIDS field
-	elseif ~isempty(strfind(sourcePar.tSequenceFileName,'CustomerSeq'))
-		bidsPar.SequenceType = 'Customer';
+    % Find out if it's a product sequence or a patch
+	if ~isempty(regexp(sourcePar.tSequenceFileName,'SiemensSeq', 'once'))
```

Comment:
Don't you want regexpi in these instances as well, like below?

Reply:
Fixed.